docs/current/quickstart/snippets/build/index.mjs

```diff
@@ -0,0 +1,31 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(async (client) => {
+
+  // use a node:16-slim container
+  // mount the source code directory on the host
+  // at /src in the container
+  const source = await client.container()
+    .from("node:16-slim")
+    .withMountedDirectory('/src', client.host().directory('.', { exclude: ["node_modules/", "ci/"] }))
```

Comment:
```suggestion
    .withMountedDirectory('/src', client.host().directory(appDirectory, { exclude: ["node_modules/", "ci/"] }))
```

Reply:
If I execute the `index.mjs` from the `ci/` directory I get the following error:
```shell
process "docker-entrypoint.sh npm install" did not complete successfully: exit code: 254\n'
...
```
Because the npm install is trying to execute from the `ci` directory. To have something more resilient, we can use the `import.meta.url` above.

Reply:
The idea of the quickstart is for users to follow it as-is. I don't think we need to be smart about how users could run things since it's indicated there that you need to run things from the repo folder.